### ISSUE TITLE
MAS 1.4.3: Elements must have sufficient color contrast (.c1)

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [jupyter/notebook#4206]
Azure Notebooks - Welcome has an accessibility issue that needs investigation.

Issue Details
Elements must have sufficient color contrast
color-contrast

Element Path
#iframe;.cell.code_cell.rendered:nth-child(4) > .input > .inner_cell > .input_area > .highlight.hl-ipython3 > pre > .c1

Snippet
<span class="c1"># hyperdrive run with child runs having metrics as series</span>

How to fix
Fix any of the following:
- Element has insufficient color contrast of 4.24 (foreground color: #408080, background color: #f7f7f7, font size: 10.5pt, font weight: normal). Expected contrast ratio of 4.5:1

To reproduce
Use K4W to investigate the issue details

Environment
Chrome version 69.0.3497.100

This accessibility issue was found using Keros for Web, a tool that helps debug and find accessibility issues earlier. Get more information & download this tool at http://aka.ms/K4Web.

[MAS 1.4.3 Elements must have sufficient color contrast (.c1)_1.zip](https://github.com/jupyter/notebook/files/2576198/MAS.1.4.3.Elements.must.have.sufficient.color.contrast.c1._1.zip)

**Comment by takluyver** [comment]
Fixed by #4746.
